components/brave_wallet_ui/page/screens/send/components/select_address_modal/select_address_modal.tsx

```diff
@@ -127,6 +128,8 @@ export const SelectAddressModal = React.forwardRef<HTMLDivElement, Props>(
       React.useState<boolean>(false)
     const [isOffChainEnsWarningDismissed, dismissOffchainEnsWarning] =
       React.useState<boolean>(false)
+    const [isGeneratingAddress, setIsGeneratingAddress] =
```

Comment:
I think you can use the loading state of the mutation instead of a new useState
```
    const [generateReceiveAddress, { isLoading: isGeneratingAddress }] =
      useGenerateReceiveAddressMutation()
```

Reply:
ahh yes, switched to this!